### ISSUE TITLE
Screen Readers Announce Time as Plain Numbers

### ISSUE THREAD

**Issue by davidsilva** [videojs/video.js#6566]
## Description
Screen readers such as JAWS and NVDA announce the current time and duration as simple numbers, e.g., "one two three" when presented with "1:23." This isn't unique to Videojs. There's no way I know of to "tell" a screen reader that "1:23" should be announced as "one minute twenty-three seconds." So, I'm wondering if there's some way to modify video.js (or use a plug-in) to change what's in the `vjs-progress-holder` element, in particular the `aria-valuetext` value to a plain-language version, e.g., "one minute twenty-three seconds of five minutes forty-five seconds." So far I haven't found a way to do this via a plug-in. Modifying the video.js code, I've only managed to get "undefined" to display in the UI where the _visual_ time indicator normally appears.

**Comment by gkatsev** [comment]
actually, going to close this in favor of https://github.com/videojs/video.js/issues/6335 because we already have an answer there. We'd like to fix it but it's _hard_.
